edpm_ansible/roles/edpm_ovn/molecule/default/prepare.yml

```diff
@@ -26,6 +26,7 @@
         test_deps_setup_stream: true
         test_deps_edpm_packages:
           - openvswitch
+          - ansible-config_template
```

Comment:
we don't use the package, we import the collection in `requirements.yml`

Reply:
Didn't know that, thanks!